### PR TITLE
[CoreMotion] Added support for Xcode 14.1 b1

### PR DESCRIPTION
**Pull request by SotoiGhost**

### CODE REVIEW

File: src/coremotion.cs [hunk 1]
```diff
@@ -859,4 +859,118 @@ interface CMRecordedPressureData {
 		NSDate StartDate { get; }
 	}
 
+	[NoMac, NoiOS, NoMacCatalyst, Watch (9,0), NoTV]
+	[Native]
+	public enum CMWaterSubmersionState : long
+	{
+		Unknown = 0,
+		NotSubmerged,
+		Submerged
```
Comment (tj-devel709) [inline src/coremotion.cs:868]:
```suggestion
		Submerged,
```

File: src/coremotion.cs [hunk 2]
```diff
@@ -859,4 +859,118 @@ interface CMRecordedPressureData {
...
+	[Native]
+	public enum CMWaterSubmersionDepthState : long
+	{
+		Unknown = 0,
+		NotSubmerged = 100,
+		SubmergedShallow = 200,
+		SubmergedDeep = 300,
+		ApproachingMaxDepth = 400,
+		PastMaxDepth = 500,
+		SensorDepthError = 600
```
Comment (tj-devel709) [inline src/coremotion.cs:881]:
```suggestion
		SensorDepthError = 600,
```

**Review by tj-devel709 — COMMENTED** [review]
A few nit things.
I also started this one, but closed it due to being on the incorrect branch and having conflicts. It did not include the 14.1 b1 changes though and can be found here: https://github.com/xamarin/xamarin-macios/pull/15977/files#diff-d27d21346ea9e21416f16996de088338485c706566c16100166f6f5d2f55d5af.
I think you'll want to add this line in as well though
https://github.com/xamarin/xamarin-macios/pull/15977/files#diff-d27d21346ea9e21416f16996de088338485c706566c16100166f6f5d2f55d5afR842

File: src/coremotion.cs [hunk 3]
```diff
@@ -859,4 +859,118 @@ interface CMRecordedPressureData {
 		NSDate StartDate { get; }
 	}
 
+	[NoMac, NoiOS, NoMacCatalyst, Watch (9,0), NoTV]
```
[the remark this answers is not in the document]
Comment (rolfbjarne) [reply src/coremotion.cs:862]: No, that's fine

**Review by mandel-macaque — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Failures are unrelated.

**Pull request merged**